min-image/runtimes/python/server.py

```diff
@@ -177,15 +185,16 @@ def main():
 
     global bootstrap_path
 
-    if len(sys.argv) < 2:
-        print("Expected execution: chroot <path_to_root_fs> python3 server.py <path_to_bootstrap.py> [cgroup-count] [enable-seccomp]")
+    if len(sys.argv) < 3:
+        print("Expected execution: chroot <path_to_root_fs> python3 server.py <path_to_bootstrap.py> [cgroup-count] [enable-seccomp] [enable-capabilities]")
         print("    cgroup-count: number of FDs (starting at 3) that refer to /sys/fs/cgroup/..../cgroup.procs files")
-        print("    enable-seccomp: true/false to enable or disables seccomp filtering")
+        print("    enable-seccomp: true/false to enable or disable seccomp filtering")
+        print("    enable-capabilities: true/false to enable or disable capability dropping")
```

Comment:
I wonder if we should call it drop-capabilities or something like that?  I think it is intuitive that "enable-seccomp" is going to restrict you.  But it's less obvious whether "enable-capabilities" is going to give you more capabilities, or turn on logic that limits your capabilities.

Reply:
That makes sense, I'll change the name